### ISSUE TITLE
Grouped/dodged bar charts not working as documented

### ISSUE THREAD

**Issue by nicolaskruchten** [holoviz/hvplot#1072]
#### ALL software version info

MacOS Big Sur, M1
Python 3.10.10
JupyterLab v3.5.2
hvplot 0.8.3
bokeh 3.1.1

#### Description of expected behavior and the observed behavior

I'm trying to make a dodged bar chart like the one in https://hvplot.holoviz.org/reference/pandas/bar.html#basic-bar-plots but the output is always stacked, never dodged

#### Complete, minimal, self-contained example code that reproduces the issue

I just downloaded this notebook https://hvplot.holoviz.org/reference/pandas/bar.html#basic-bar-plots and did run-all:

<img width="1112" alt="image" src="https://github.com/holoviz/hvplot/assets/203523/3b34f6c6-c17a-4e7e-b345-952efecbe695">

No errors in the console.

**Comment by philippjfr** [comment]
Indeed, seems to be a regression in Bokeh 3.x. See also https://github.com/holoviz/holoviews/issues/5726
